providers/clicksend/src/lib/clicksend.provider.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { ClicksendSmsProvider } from './clicksend.provider';
+
+describe('ClicksendSMSProvider', () => {
+  let provider: ClicksendSmsProvider;
+
+  beforeEach(() => {
+    provider = new ClicksendSmsProvider({
+      username: '<your-clicksend-username>',
+      apiKey: '<your-clicksend-APIkey>',
+    });
+  });
+
+  it('should be defined', () => {
+    expect(provider).toBeDefined();
+  });
+
+  describe('sendMessage', () => {
+    it('should return a response with an id and date', async () => {
+      const response = await provider.sendMessage({
+        to: '+0451111111',
+        content: 'test message',
+      });
+
+      expect(response).toHaveProperty('id');
```

Comment:
You'll need to mock the response to make the tests pass.
For reference check `clickatell` providers spec file, how the sendMessage method is mocked.

Reply:
Hey @BiswaViraj! I've updated the spec file by mocking the response. The tests passed as well.
Please let me know if there's anything else required from my side :)

![WhatsApp Image 2023-10-11 at 19 43 50_6e90890d](https://github.com/novuhq/novu/assets/97528080/2ebb50bb-21df-4b36-87e1-c4efb9e3a99d)

Reply:
Thank you, merging it now